loader/loader.go

```diff
@@ -212,8 +215,10 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, runFatalCh
 				})
 				continue
 			}
+			txnHistogram.WithLabelValues(w.cfg.Name, w.cfg.WorkerName, w.cfg.SourceID, job.schema, job.table).Observe(time.Since(startTime).Seconds())
```

Comment:
not sure when disk IO is slow, writing log will increase the time cost of this metric

Reply:
i don;t think so, this metric only observes the time taken to execute sqls.( startTime on L199

Reply:
in `w.conn.executeSQL`, it may write some log

Reply:
maybe we can use async logger?